test/athens/common_events/atomic_ops/block_save_test.cljc

```diff
@@ -102,3 +105,59 @@
         (t/is (= new-str (common-db/v-by-ea @@fixture/connection
                                             child-1-eid :block/string)))))))
 
+
+(t/deftest undo
+  (let [test-uid "test-uid"
+        get-str  (fn []
+                   (->> [:block/uid test-uid]
+                        (common-db/get-internal-representation @@fixture/connection)
+                        :block/string))
+        setup!   (fn [string]
+                   (->> [{:page/title     "test-page"
```

Comment:
Pages aren't cleared between tests, we need either teardown or randomized names, or something deterministic.